charts/charts/tenant-namespace-operator/values.yaml

```diff
@@ -56,3 +56,7 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+ingressClass:
+  enabled: true
+  name: nginx
```

Comment:
This default name could be matched to the default name `public` in the tenant-namespace chart at https://github.com/pnnl-miscscripts/miscscripts/blob/master/charts/charts/tenant-namespace/values.yaml#L74